docs/docs/configuration.md

```diff
@@ -30,66 +34,66 @@ Here is an overview of all the configurable options. Keep scrolling for more inf
 ## Prefix
 Usage: `config prefix` or `config prefix [prefix]`
 
-You can either view or set the custom prefix to be used in your guild. If at any time you forget the prefix, simply mention the bot `@Suggestions#2602` or do `@Suggestions#2602 prefix`.
+You can either view or set the custom prefix to be used in your server. If at any time you forget the prefix, simply mention the bot `@Suggestions#2602` or run `@Suggestions#2602 prefix`.
 
 The bot's mention can also be used as a command prefix.
 
 ## Suggestions Channel
 Usage: `config channel` or `config channel [channel]`
 
-You can either view or set the channel where suggestions will be posted to. Invalid channels will not be allowed to be set. The options to pass through when setting the channel can be:
+You can either view or set the channel where suggestions will be posted. Invalid channels can't be set. The options to pass through when setting the channel can be:
 
-- The channel mention - `#suggestions`
-- The channel name - `suggestions`
-- The channel ID - `480265871596584970`
+- the channel mention - `#suggestions`
+- the channel name - `suggestions`
+- the channel ID - `480265871596584970`
 
 ## Suggestions Logs Channel
 Usage: `config logs` or `config logs [channel]`
 
-You can either view or set the channel where suggestion results will be posted after a suggestion is approved or denied. The options to pass through follow the same as setting the [Suggestions Channel](#suggestions-channel). Keep in mind you cannot set a channel that does not exist.
+You can either view or set the channel where suggestion results will be posted after a suggestion is approved or denied. The options to pass through follow the same as setting the [Suggestions Channel](#suggestions-channel). Keep in mind you can't set a channel that doesn't exist.
 
 ## Staff Suggestions Channel
 Usage: `config staffchannel` or `config staffchannel [channel]`
 
-You can either view or set the channel where staff suggestions will be posted to. The options to pass through follow the same as setting the [Suggestions Channel](#suggestions-channel). Keep in mind you cannot set a channel that does not exist.
+You can either view or set the channel where staff suggestions will be posted. The options to pass through follow the same as setting the [Suggestions Channel](#suggestions-channel). Keep in mind you can't set a channel that doesn't exist.
```

Comment:
I would even extend this by also saying you can't set a channel that doesn't exist _in the server_.

Reply:
Done